js/genform.js

```diff
@@ -180,3 +180,16 @@ function checkContactValidity(el) {
         el.classList.add('is-danger');
     }
 }
+
+function autoprefixEmail(el) {
+    // Add mailto: prefix, e.g. contact@example.com -> mailto:contact@example.com
+
+    // NOTE: If this regular expression is changed in future, it should always fail if
+    // the user enters some other schema (e.g. https://) so they always have the option of overriding it.
+    // Currently this is achieved by not allowing a colon to appear in the input.
+    if (/^[a-z0-9_+.-]+@[a-z0-9_+.-]+$/i.test(el.value)) {
+        el.value = 'mailto:' + el.value;
+    }
+
+    checkContactValidity(el); // potentially remove validation error if issue resolved
```

Comment:
I personally don't like such short variable names that do not describe anything (`el`).

Reply:
Agreed, I've pushed a commit to fix this - thanks for catching that